data/json/items/comestibles/raw_veggy.json

```diff
@@ -1108,7 +1108,7 @@
     "symbol": "%",
     "quench": 6,
     "calories": 25,
-    "description": "Some young japanese knotweed stems.  They taste a bit like rhubarb and contain good amounts of vitamin A.  Refreshing.",
+    "description": "Some young Japanese knotweed stems.  They taste a bit like rhubarb and contain good amounts of vitamin A.  Refreshing.",
```

Comment:
Not 100% sure about this but I am no native speaker.

Reply:
Japanese is a proper noun